### PR TITLE
Add project files

### PR DESCRIPTION
**Pull request by vladanvi99**
In this pull request, I will add project HTML, CSS, and JS files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (vichuge) [inline README.md]: - [ ] The README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 🐶

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (vichuge) [inline README.md]: - [ ] The license should be [Creative Commons License](https://creativecommons.org/licenses/) in place for MIT. 🐱

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+## Built With
+
+- HTML
+- CSS
+- SCSS
+- JavaScript
+
+## Live Demo
+
+No Live Preview
```
Comment (vichuge) [inline README.md]: - [ ] Please, add a Live demo made with GitHub pages on your readme file. Configure it to use the branch where you're working now (you can do it con the configuration section on a project on GitHub page). 🐦

File: about.html [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+        <!--HEADER START-->
+        <div class="circle-bg"></div>
+        <header class="header-about">
+            <nav>
+                <a href="index.html" class="logo">
+                    <img src="./images/global/logoNoTextBlack.png" alt="logo">
+                    NIKE
+                </a>
+                <button type="button" class="open-mobile-menu">
+                    <img src="./images/global/menu-toggler.svg" alt="open-mobile-menu">
```
Comment (vichuge) [inline about.html]: - [ ] I invite you to use another icon, a burger icon are 3 horizontal lines. Use the burger icon is more understandable for the most part users. If you want to have more info about burger icon [check this link](https://en.wikipedia.org/wiki/Hamburger_button). 🐊 
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    <link rel="icon" href="./images/global/favIcon.png">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+</head>
+<body>
+    <div class="wrap red-color">
+        <a href="#" class="back-to-top-btn"><i class="fas fa-chevron-up"></i></a>
+        <!--HEADER START-->
+        <div class="circle-bg"></div>
+        <header class="header-about">
+            <nav>
```
Comment (vichuge) [inline about.html:19]: - [ ] The navbar should be separate in both, top dark bar and the bottom white bar. Your design is some! now I kindly invite you to change this and other details more about the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 🐤 
image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+        <!--CONTACT END-->
+        <!--FOOTE START-->
+        <footer>
+            <div class="footer-top">
+
+            </div>
+            <div class="footer-bottom">
+                <p>CopyRight&COPY; By <a href="https://www.facebook.com/ITHive-Despotovac-101366482012645" rel="noopener noreferrer" target="_blank">Vladan Videnovic</a></p>
+            </div>
+        </footer>
```
Comment (vichuge) [inline about.html]: - [ ] The footer should be more like the [given design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 🦊 
- A logo center to the left and text on the right side.
- Dark color for desktop
image

- White color for mobile
image

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] On video presentation, I invite you to add a highlight about a piece of the code you find interesting or proud to make.
- Again, I love you design, but according to given template, it has a lot of things to improve:
- [ ] The pages should look almost identical to the original design, outside the content differences.
- [ ] The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.
- Basic color swatch: #272A31, #EC5242, #D3D3D3
- [ ] The pages use the same fonts or similar fonts from the original design.
- Fonts used originally: Cocogoose, Lato
- [ ] All page elements have similar spacing, borders, shadows, and paddings to the original design.

| original | your design |
| --- | --- |
| image | image |
| image | image |
| image | image |

- [ ] The form section is not on the template, I kindly invite you to remove it.
image

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 5 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @vladanvi99🔥 

You did excellent work as always, nothing more to add here!

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**